ReactNative/ReactTwitter/core/views/tweetsListItem.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import {
+  Alert,
+  StyleSheet,
+  View,
+  Image,
+  Text,
+  TouchableHighlight
+} from 'react-native'
+
+var TweetsListItem = React.createClass({
+  propTypes: {
+    id: React.PropTypes.string.isRequired,
+    time: React.PropTypes.number.isRequired,
+    author_avatar: React.PropTypes.string.isRequired,
+    author_name: React.PropTypes.string.isRequired,
+    author_handle: React.PropTypes.string.isRequired,
+    text: React.PropTypes.string.isRequired
+  },
+
+  render () {
+    var formattedTime = this._formatTime(this.props.time)
+    return (
+      <TouchableHighlight onPress={() => this._tweetSelected(this.props.id)}
+        underlayColor='#dddddd'>
+         <View>
+          <View style={styles.rowContainer}>
+            <Image style={styles.tweet_avatar} source={{ uri: this.props.author_avatar }} />
+            <View style={styles.textContainer}>
+              <Text style={styles.tweet_author}>{this.props.author_name}</Text>
+              <Text style={styles.tweet_author_handle}>@{this.props.author_handle}</Text>
+              <Text style={styles.tweet_time}>{formattedTime}</Text>
+              <Text style={styles.tweet_text}>{this.props.text}</Text>
+            </View>
+           </View>
+           <View style={styles.separator}/>
+         </View>
+       </TouchableHighlight>
+    )
+  },
+
+  _formatTime (time) {
+    var dateFormat = require('dateformat')
+    return dateFormat(time, 'mmmm dS, yyyy, h:MM:ss TT')
+  },
+
+  _tweetSelected (tweetId) {
+    // TODO: navigate to tweet details screen
+    Alert.alert(
```

Comment:
It's interesting to see which iOS component is being used for the alert as we have a couple of them and one is going to be deprecated